nspanel_blueprint.yaml

```diff
@@ -3570,417 +3570,554 @@ variables:
+          on: zapnuté
+          off: vypnuté
+        heat: kúrenie
+      please_confirm: Potvrďte, prosím
+    SVN:
+      weekdays:
+        mon: Ponedeljek
+        tue: Torek
+        wed: Sreda
+        thu: Četrtek
+        fri: Petek
+        sat: Sobota
+        sun: Nedelja
+      today: Danes
+      tomorrow: Jutri
+      in_2_days: v 2 dneh
+      in_3_days: v 3 dneh
+      in_4_days: v 4 dneh
+      climate:
+        states:
+          on: na
+          off: izklop
+        heat: toplota
+      please_confirm: Prosimo, potrdite
+    SWE:
+      weekdays:
+        mon: Måndag
+        tue: Tisdag
+        wed: Onsdag
+        thu: Torsdag
+        fri: Fredag
+        sat: Lördag
+        sun: Söndag
+      today: Idag
+      tomorrow: Imorgon
+      in_2_days: om 2 dagar
+      in_3_days: om 3 dagar
+      in_4_days: om 4 dagar
+      climate:
+        states:
+          on: på
+          off: av
+        heat: värme
+      please_confirm: Vänligen bekräfta
+    TUR:
+      weekdays:
+        mon: Pazartesi
+        tue: Salı
+        wed: Çarşamba
+        thu: Perşembe
+        fri: Cuma
+        sat: Cumartesi
+        sun: Pazar
+      today: Bugün
+      tomorrow: Yarın
+      in_2_days: 2 Gün İçinde
+      in_3_days: 3 Gün İçinde
+      in_4_days: 4 Gün İçinde
+      climate:
+        states:
+          on: üzerinde
+          off: kapalı
+        heat: ısı
+      please_confirm: Lütfen onaylayın
+    UKR:
+      weekdays:
+        mon: Понеділок
+        tue: Вівторок
+        wed: середа
+        thu: четвер
+        fri: п ятниця
+        sat: Субота
+        sun: Неділя
+      today: Сьогодні
+      tomorrow: Завтра
+      in_2_days: через 2 дні
+      in_3_days: через 3 дні
+      in_4_days: через 2 дні
+      climate:
+        states:
+          on: увімк
+          off: вимк
+        heat: нагрів
+      please_confirm: Будь ласка, підтвердіть
+  mui_weekday_today:     "{{ mui[language].weekdays[['mon','tue','wed','thu','fri','sat','sun'][now().weekday()]]                       ~ ', ' ~ as_timestamp(now())                     | timestamp_custom('%d.%m') }}"
+  mui_weekday_tomorrow:  "{{ mui[language].weekdays[['mon','tue','wed','thu','fri','sat','sun'][(now() + timedelta(days=1)).weekday()]] ~ ', ' ~ as_timestamp(now() + timedelta(days=1)) | timestamp_custom('%d.%m') }}"
+  mui_weekday_in_2_days: "{{ mui[language].weekdays[['mon','tue','wed','thu','fri','sat','sun'][(now() + timedelta(days=2)).weekday()]] ~ ', ' ~ as_timestamp(now() + timedelta(days=2)) | timestamp_custom('%d.%m') }}"
+  mui_weekday_in_3_days: "{{ mui[language].weekdays[['mon','tue','wed','thu','fri','sat','sun'][(now() + timedelta(days=3)).weekday()]] ~ ', ' ~ as_timestamp(now() + timedelta(days=3)) | timestamp_custom('%d.%m') }}"
+  mui_weekday_in_4_days: "{{ mui[language].weekdays[['mon','tue','wed','thu','fri','sat','sun'][(now() + timedelta(days=4)).weekday()]] ~ ', ' ~ as_timestamp(now() + timedelta(days=4)) | timestamp_custom('%d.%m') }}"
```

Comment:
This code below is repeated multiple times.
`['mon','tue','wed','thu','fri','sat','sun']`

It should be possible to have this as a variable.

Reply:
Like:
```yaml
mui_weekdays:
   - mon
   - tue
   - wed
   - thu
   - fri
   - sat
   - sun

mui_weekday_today:     "{{ mui[language].weekdays[mui_weekdays[now().weekday()]]                       ~ ', ' ~ as_timestamp(now())                     | timestamp_custom('%d.%m') }}"
mui_weekday_tomorrow:  "{{ mui[language].weekdays[mui_weekdays[(now() + timedelta(days=1)).weekday()]] ~ ', ' ~ as_timestamp(now() + timedelta(days=1)) | timestamp_custom('%d.%m') }}"
```

Reply:
Good point. I think I can remove that list entirely.
I will take a look later.